_toc.yml

```diff
@@ -375,6 +375,8 @@ entries:
             - file: docs/products/kafka/kafka-connect/howto/mongodb-sink-lenses
             - file: docs/products/kafka/kafka-connect/howto/redis-streamreactor-sink
             - file: docs/products/kafka/kafka-connect/howto/cassandra-streamreactor-sink
+            - file: docs/products/kafka/kafka-connect/howto/gcp-pubsub-sink
+            - file: docs/products/kafka/kafka-connect/howto/gcp-pubsub-lite-sink
```

Comment:
Nitpicking so feel free to ignore but the organisation is not in alphabetical order, do we want it to be?

Reply:
I believe we can review the sorting at a later stage?
I created an issue for this #1525